### PR TITLE
Add snapshot_info jsonrpc method

### PR DESCRIPTION
**Pull request by madninja**

### CODE REVIEW

**Review by Vagabond — APPROVED** [review]

File: src/jsonrpc/miner_jsonrpc_snapshot.erl [hunk 1]
```diff
@@ -14,13 +14,29 @@
...
+                    height => Height,
+                    hash => ?BIN_TO_B64(Hash),
+                    hash_hex => blockchain_utils:bin_to_hex(Hash),
+                    have_snapshot => have_snapshot(Hash, Chain)
+                }
+             || {Height, _, Hash} <- Snapshots
+            ]
+    end;
+handle_rpc(<<"snapshot_info">>, [#{<<"path">> := Path}]) ->
+    case file:read_file(Filename) of
```
Comment (jadeallenx) [inline src/jsonrpc/miner_jsonrpc_snapshot.erl:31]: where does this filename come from? it doesn't seem to be bound in the parameters?

Comment (abhay) [inline src/jsonrpc/miner_jsonrpc_snapshot.erl:31]:
```suggestion
    case file:read_file(Path) of
```

File: src/jsonrpc/miner_jsonrpc_snapshot.erl [hunk 2]
```diff
@@ -14,13 +14,29 @@
...
+        {error, Error} ->
+            ?jsonrpc_error({error, Error});
+        {ok, Snap} ->
```
Comment (abhay) [inline src/jsonrpc/miner_jsonrpc_snapshot.erl:34]:
```suggestion
        {ok, BinSnap} ->
```

File: src/jsonrpc/miner_jsonrpc_snapshot.erl [hunk 3]
```diff
@@ -14,13 +14,29 @@
...
+            blockchain_ledger_snapshot_v1:deserialize(BinSnap),
```
Comment (abhay) [inline src/jsonrpc/miner_jsonrpc_snapshot.erl:35]:
```suggestion
            blockchain_ledger_snapshot_v1:deserialize(Snap),
```

**Review by abhay — APPROVED** [review]

**Pull request merged**